README.md

```diff
@@ -254,6 +255,46 @@ export default class CounterApp {
 }
 ```
 
+#### Selectors
+
+Selectors let you define views on your state. They enable you to define derived data on your store's state. 
+In combination with memoized functions the calculation overhead can be minmized. Hence, the child components of a Connector 
+ using memoized selectors will only be rerendered if the source data of the selector changes. This can help to
+ prevent store dependencies.
+ 
+It is further recommended to define complex selectors in separate modules.
+
+e.g. defining selectors for a Todo Store
+```js
+import { createSelector, createBuffered } from 'redux';
+
+export let todoSelector = createSelector('todos');
+export let numberOfTodos = createSelector(todoSelector, createBuffered(todos => todos.length)); 
```

Comment:
This may have been discussed, or I may be missing something obvious here, but why separate creating selectors from memoization?